README.md

```diff
@@ -1,23 +1,21 @@
 <h1 align="center"> <a href="https://www.jsdelivr.com/globalping"><img width="28" alt="Globalping icon" src="https://user-images.githubusercontent.com/1834071/216975126-01529980-a87e-478c-8ab3-bf7d927a1986.png"></a> Globalping CLI </h1>
 
-<p align="center">Access a global network of probes without leaving your console. Powered by the Globalping community!</p>
+<p align="center">Run measurement commands from a global network of probes right from your console with the official command-line interface for <a href="https://github.com/jsdelivr/globalping">Globalping</a>. Powered by the Globalping community!</p>
 
 <p align="center"><img width="80%" src="https://user-images.githubusercontent.com/1834071/217010016-9da38f12-906a-47cf-adca-18017588efe5.png">
 </p>
 
-- The official command-line interface for the [Globalping](https://github.com/jsdelivr/globalping) network.
+## Key features
 - Run networking commands from any location in the world
-- Supported commands: ping, mtr, traceroute, dns resolve, HTTP
-- Includes detailed timings and latency metrics with every test
-- Real-time results right in your command line
-- Human friendly format and output
-- Cross-platform. Linux, MacOS, Windows are all supported
-- Auto-updated via all automated installation methods
-- [Check our website for online tools, our Slack app and more!](https://www.jsdelivr.com/globalping)
+- Real-time results for ping, mtr, traceroute, DNS resolve, HTTP
+- Detailed timings and latency metrics
+- Human-friendly format and output
+- Supports Linux, MacOS, and Windows
+- Auto-updated through package managers
+- Explore additional [Globalping integrations](https://www.jsdelivr.com/globalping), including our online tools, Slack app, and more
```

Comment:
Why make them shorter? It feels like we lost information and value by doing so

Reply:
I reworked the list - let me know if this looks better to you.